### PR TITLE
Temp bu

### PR DESCRIPTION
**Pull request by saiprasad1997**

### CODE REVIEW

File: e2e/src/app.e2e-spec.ts [hunk 1]
```diff
@@ -24,15 +25,152 @@ describe('workspace-project App', () => {
...
+    expect(buPage.enterBuStatus('Deactivated'));
+    expect(buPage.clickValidateRegisterButton());
+  });
+
+  it('should set partner fields and go to onboard inconatct BU page', function () {
+    expect(buPage.enterBuName('Ring Central' + new Date().getTime()));
+    expect(buPage.enterBuId('12' + new Date().getTime()));
+    expect(buPage.enterPartnerId('4'));
+    expect(buPage.enterUsername('admin@RingCentral.com' + new Date().getTime()));
+    console.log('12' + new Date().getTime());
```
Comment (KMayuri25) [inline e2e/src/app.e2e-spec.ts]: Remove this

File: e2e/src/app.e2e-spec.ts [hunk 2]
```diff
@@ -24,15 +25,152 @@ describe('workspace-project App', () => {
...
+describe('BU onboard incontact Test Cases', () => {
+  let buIcPage: BuIcPage;
+
+  beforeEach(() => {
+    buIcPage = new BuIcPage();
+  });
+
+
+  it('should check if incontact fields are present', function () {
+    // expect(buIcPage.navigateTo());
```
Comment (KMayuri25) [inline e2e/src/app.e2e-spec.ts]: Remove commented code

File: e2e/src/app.po.ts [hunk 3]
```diff
@@ -86,3 +86,296 @@ export class AppPage {
...
+    return this.getIcPasswordTextBox().sendKeys(password);
+  }
+
+  enterIcBuStatus(bustatus) {
+    this.getIcBuStatusTextBox().clear();
+    return this.getIcBuStatusTextBox().sendKeys(bustatus);
+  }
+
+
+ /*getBuNameFieldLoaded() {
```
Comment (KMayuri25) [inline e2e/src/app.po.ts]: Remove commented

File: src/app/bu/create-ic-bu/component/create-ic-bu.component.spec.ts [hunk 4]
```diff
@@ -133,12 +132,12 @@ describe('CreateIcBuComponent', () => {
   });
 
 
-  it('should route to bu/map after onboarding partner BU', () => {
+  /*it('should route to bu/map after onboarding partner BU', () => {
```
Comment (KMayuri25) [inline src/app/bu/create-ic-bu/component/create-ic-bu.component.spec.ts]: Remove comments

File: e2e/src/app.e2e-spec.ts [hunk 5]
```diff
@@ -24,12 +24,144 @@ describe('workspace-project App', () => {
     expect(page.getAlertMessage()).toEqual('Username or password or platform is incorrect');
   });
   it('should enter valid credentials, hit login button and landing page should be displayed', () => {
-    expect(page.enterUsername('amarp@TC4.incontact.com'));
-    expect(page.enterPassword('Pune@4321'));
+    expect(page.enterUsername('devadmin@tc3.com'));
```
Comment (abhilash-incontact) [inline e2e/src/app.e2e-spec.ts]: Put these credential in some config file and read them from there.

File: e2e/src/app.e2e-spec.ts [hunk 6]
```diff
@@ -24,12 +24,144 @@ describe('workspace-project App', () => {
...
+    expect(buIcPage.enterIcBuId('A2'));
+    expect(buIcPage.enterIcPartnerId('4'));
+    expect(buIcPage.enterIcUsername('admin@RingCentral'));
+    expect(buIcPage.enterIcPassword('asdfghjk'));
+    expect(buIcPage.enterIcBuStatus('Deactivated'));
+    expect(buIcPage.isICValidateRegisterButtonPresent());
+    expect(buIcPage.clickICValidateRegisterButton());
+  });
+
+  it('display validation messages on entering invalid data 2', function () {
```
Comment (abhilash-incontact) [inline e2e/src/app.e2e-spec.ts]: I don't see any error message check. All 3 test look same ?

**Pull request merged**